keras_nlp/models/bert/bert_backbone.py

```diff
@@ -163,6 +163,7 @@ def __init__(
                     x, approximate=True
                 ),
                 dropout=dropout,
+                epsilon=1e-12,
```

Comment:
@soma2000-lang the param name is `layer_norm_epilson` so test are failing. Can you change?

Reply:
Sure